pipeline-templates/step/deploy.yml

```diff
@@ -0,0 +1,31 @@
+parameters:
+  VerifyDeployment: false
```

Comment:
This doesn't need a default value here (unlike the building block) as we should be making a deliberate choice depending on the environment

Reply:
Good spot, fixed